src/drivers/Kconfig

```diff
@@ -6,4 +6,16 @@ source "src/drivers/intel/cavs/Kconfig"
 
 source "src/drivers/dw/Kconfig"
 
+config DUMMY_DMA
+	bool "Dummy DMA (software DMA driver)"
```

Comment:
Shouldn't this entry be in drivers/generic/Kconfig ?

Reply:
For now I'll leave it global. Maybe when a second generic driver will come this can be moved.